test/ResizerTests.cs

```diff
@@ -0,0 +1,153 @@
+using System;
+using System.IO;
+using Xunit;
+using SixLabors.ImageSharp;
+
+namespace ImageResizer.Tests
+{
+    public class ResizerTests
+    {
+        private Resizer _resizer;
+
+        public ResizerTests()
+        {
+            _resizer = new Resizer();
```

Comment:
can we have this be a local var in each of the tests? sorry, just a personal preference to keep things more self-contained